services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> list[Movie]:
+    queryset = Movie.objects.all()
+    if None not in (genres_ids, actors_ids):
+        return queryset.filter(
+            actors__id__in=actors_ids,
+            genres__id__in=genres_ids,
+        )
+    elif genres_ids is not None:
+        return queryset.filter(genres__id__in=genres_ids)
+    elif actors_ids is not None:
+        return queryset.filter(actors__id__in=actors_ids)

```

Comment:
So hard code. You just have two cases:
1. if genres_ids is not None
2. if actors_ids is not None